tests/persistence/test_persistence_leverage.py

```diff
@@ -0,0 +1,638 @@
+from datetime import datetime, timedelta
+from math import isclose
+
+import pytest
+
+from freqtrade.enums import InterestMode
+from freqtrade.persistence import Trade
+from tests.conftest import log_has_re
+
+
+@pytest.mark.usefixtures("init_persistence")
+def test_interest_kraken_lev(market_lev_buy_order, fee):
+    """
+        Market trade on Kraken at 3x and 5x leverage
+        Short trade
+        interest_rate: 0.05%, 0.25% per 4 hrs
+        open_rate: 0.00004099 base
+        close_rate: 0.00004173 base
+        stake_amount: 0.0037707443218227
+        borrowed: 0.0075414886436454
+        amount:
+            275.97543219 crypto
+            459.95905365 crypto
+        borrowed:
+            0.0075414886436454  base
+            0.0150829772872908  base
+        time-periods: 10 minutes(rounds up to 1 time-period of 4hrs)
+                        5 hours = 5/4
+
+        interest: borrowed * interest_rate * ceil(1 + time-periods)
+                = 0.0075414886436454 * 0.0005  * ceil(2)   = 7.5414886436454e-06 base
+                = 0.0075414886436454 * 0.00025 * ceil(9/4) = 5.65611648273405e-06 base
+                = 0.0150829772872908 * 0.0005  * ceil(9/4) = 2.26244659309362e-05 base
+                = 0.0150829772872908 * 0.00025 * ceil(2)   = 7.5414886436454e-06 base
+    """
+
+    trade = Trade(
+        pair='ETH/BTC',
+        stake_amount=0.0037707443218227,
+        amount=275.97543219,
```

Comment:
I think this would be a lot simpler by using "USDT" pairs (simply because our mind calculations work better with numbers > 0).
it'll make the calculation easier (as the result will not be 1e-06).

Also - the precision used above is unrealistic - no exchange will allow you 10 decimals on any number (the maximum is usually 8 for BTC pairs, and 3-4 for USDT pairs iirc).

THis also makes "recalculating" this with the [kraken fee calculator](https://kraken-fees-calculator.github.io/) impossible - as they don't allow these numbers.